### DISCUSSION TITLE
Server Permissions?

### DISCUSSION THREAD

**Discussion by espartiskills** [kous500/CurveBuilding#34] [Q&A] [answered]
Is there any way to grant a user permission to use the //bcpos and //bc commands without giving them admin/op? They have worldedit perms, but still can't use the CurbeBuilding commands

**Comment by kous500** [comment, marked as answer]
For the Bukkit version: You can manage permissions using a plugin like LuckPerms. To allow a user to use CurveBuilding commands, you can grant these specific permissions:

- `CurveBuilding.pos`: Allows use of the `/bcpos` command.
- `CurveBuilding.bc`: Allows use of the `/bc` command.
- `CurveBuilding.admin`: Grants full access to all CurveBuilding commands.

For the Fabric version: Currently, to run CurveBuilding commands, users need op level 2 or higher. There isn't a permissions management system yet. I plan to improve this by allowing creative mode users to execute these commands in future updates.

**Reply by espartiskills** [reply]
Yeah, I'm running on a Fabric Server at the moment, so that would explain it. It's a whitelist server, so we're not too worried about giving people access to worldEdit.
The plugin is unbelievably amazing in singleplayer, so I look forward to being able to use it on a Fabric server soon. Thanks!
